### PR TITLE
Adding scope to GKE clusters for boskos

### PR DESCRIPTION
**Pull request by utka**

### CODE REVIEW

File: boskos/gcp/test-configs.yaml [hunk 1]
```diff
@@ -20,6 +20,7 @@ configs:
             - https-server
             scopes:
             - https://www.googleapis.com/auth/cloud-platform
+            - https://www.googleapis.com/auth/trace.append
```
Comment (sebastienvas) [inline boskos/gcp/test-configs.yaml:26]: dont you need to add under cluster ?

File: boskos/gcp/test-configs.yaml [hunk 2]
```diff
@@ -44,3 +45,4 @@ configs:
...
```
Comment (sebastienvas) [inline boskos/gcp/test-configs.yaml:55]: same comment here

File: boskos/gcp/config_test.go [hunk 3]
```diff
@@ -51,6 +51,7 @@ func TestParseInvalidConfig(t *testing.T) {
 					},
 					Scopes: []string{
 						"https://www.googleapis.com/auth/cloud-platform",
+						"https://www.googleapis.com/auth/trace.append",
```
Comment (sebastienvas) [inline boskos/gcp/config_test.go:58]: Don't you need to add this under cluster ?

File: boskos/gcp/config_test.go [hunk 4]
```diff
@@ -78,6 +79,7 @@ func TestParseInvalidConfig(t *testing.T) {
...
```
Comment (sebastienvas) [inline boskos/gcp/config_test.go:90]: Same comment here.

**Review by sebastienvas — CHANGES_REQUESTED** [review]
the code change seems fine, but you added the scopes in the vms. You should add them in the Cluster, and verify that in test.

File: boskos/gcp/config_test.go (hunk 3 above)
Comment (utka) [reply boskos/gcp/config_test.go:58]: You are right! Good catch

File: boskos/gcp/config_test.go (hunk 4 above)
Comment (utka) [reply boskos/gcp/config_test.go:90]: Done

File: boskos/gcp/test-configs.yaml (hunk 1 above)
Comment (utka) [reply boskos/gcp/test-configs.yaml:26]: Done

File: boskos/gcp/test-configs.yaml (hunk 2 above)
Comment (utka) [reply boskos/gcp/test-configs.yaml:55]: Done

**Comment by utka** [comment]
@sebastienvas ptal

**Comment by sebastienvas** [comment]
You need to release another version of Mason for this to take effect.

**Comment by sebastienvas** [comment]
https://github.com/istio/test-infra/blob/master/boskos/Makefile#L8 and then update https://github.com/istio/test-infra/blob/master/boskos/mason-deployment.yaml#L18, and make mason-depoyment

**Pull request merged**